### PR TITLE
rook: Update to 1.4.4 to use raw mode

### PR DESCRIPTION
**Pull request by kfyharukz**
This PR bump the Rook version to 1.4.4
It also includes the test to reconstruct `ceph-ssd` CephCluster.
The updating process takes a too long time so we extend the timeout for apps synchronization.

### CODE REVIEW

File: test/setup_test.go [hunk 1]
```diff
@@ -199,6 +223,12 @@ func testSetup() {
 		applyAndWaitForApplications(commitID)
 	})
 
+	// This block is for workaround to avoid tls handshake error due to deletion ceph-ssd-block-pool
+	// Please delete this block after Rook 1.4 is applied
```
Comment (morimoto-cybozu) [inline test/setup_test.go:227]: Use "TODO:" for ease of search.
```suggestion
	// TODO: delete this block after Rook 1.4 is applied
```

File: test/setup_test.go [hunk 2]
```diff
@@ -364,15 +383,18 @@ func applyAndWaitForApplications(commitID string) {
 	}
 	// want to do "Eventually( Consistently(checkAllAppsSynced, 30sec, 1sec) )"
```
Comment (morimoto-cybozu) [inline test/setup_test.go:384]: Update this comment too.
I'm not sure if the duration of 15 seconds is long enough to examine the stability.  The original setting of 30 seconds did not have a concrete reason, though.

```suggestion
	// want to do "Eventually( Consistently(checkAllAppsSynced, 15sec, 1sec) )"
```

File: docs/maintenance.md [hunk 3]
```diff
@@ -129,17 +129,16 @@ See: [Kubernetes controllers configuration](https://docs.projectcalico.org/refer
 
 *Do not upgrade Rook and Ceph at the same time!*
 
-Read [this document](https://github.com/rook/rook/blob/master/Documentation/ceph-upgrade.md) before.
+Read [this document](https://github.com/rook/rook/blob/master/Documentation/ceph-upgrade.md) before. Note that you should choose the appropriate release version.
```
Comment (morimoto-cybozu) [inline docs/maintenance.md:132]:
```suggestion
Read [this document](https://github.com/rook/rook/blob/master/Documentation/ceph-upgrade.md) first. Note that you should choose the appropriate release version.
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: test/setup_test.go (hunk 2 above)
Comment (kfyharukz) [reply test/setup_test.go:384]: The command `argocd app get` itself takes 1 or 2 sec, so actually this process consumes a long time (e.g., over 10 min).  I think it is enough time to confirm convergence.

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Review by satoru-takeuchi — DISMISSED** [review]

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**